.github/workflows/e2e.ci.yaml

```diff
@@ -28,8 +28,10 @@ env:
   SKIP_TEST: true
   ISTIO_VERSION: 1.7.1
   TAG: ${{ github.sha }}
-  SCRIPTS_DIR: main/test/e2e-mesh/e2e-istio/scripts
-  SW_OAP_BASE_IMAGE: openjdk:8-jre-alpine
+  OAP_TAG: ${{ github.sha }}
```

Comment:
Who provides this `github.sha`?

Reply:
> Who provides this `github.sha`?

It's built in

Reply:
I mean this is `skywalking-kubernetes` repo, why this sha code could be used for OAP_TAG?

Reply:
> I mean this is `skywalking-kubernetes` repo, why this sha code could be used for OAP_TAG?

It's used as the tag when building a new OAP docker.

Though I ask to replace with a pre-built Docker image https://github.com/apache/skywalking-kubernetes/pull/83#discussion_r755736325

Reply:
Oh, yes, using pre-built makes more sense.

Reply:
Change to using the snapshot version, no need to use build-in `sha` again.